tests/unit/CMakeLists.txt

```diff
@@ -439,3 +439,7 @@ target_link_libraries(${test_prefix}pretty_print_ast_to_original_expression_test
 # Tests for mg-coordinator
 add_unit_test(coordinator_shard_map.cpp)
 target_link_libraries(${test_prefix}coordinator_shard_map mg-coordinator)
+
+# Tests for Histogram
+add_unit_test(histogram.cpp)
+target_link_libraries(${test_prefix}histogram mg-utils)
```

Comment:
Two things here:
1. There is a section started by comment `# Test mg-utils`, put this test there.
2. for the unit tests we prefix the files with the library name is most cases. E.g. this file should be `utils_historgram.cpp`. We might missed this recently, but I think we should still stick to this.

Reply:
thanks, I've renamed the test and moved its position in the cmake file to be in-line with other utils tests